routes/forecast.js

```diff
@@ -0,0 +1,46 @@
+var express = require('express');
```

Comment:
Looks good, man.  I think you can more your require('dotenv').config() into app.js, that way you don't have to require it in each individual router file.